runtime/datetime.c

```diff
@@ -103,13 +103,11 @@ timeval2syslogTime(struct timeval *tp, struct syslogTime *t, const int inUTC)
 	struct tm tmBuf;
 	long lBias;
 	time_t secs;
-/* AIXPORT : fix build error : "tm_gmtoff" is not a member of "struct tm"
- *           Choose the HPUX code path, only for this function.
- *           This is achieved by adding a check to _AIX wherever _hpux is checked
+/* AIXPORT : bug fix: manage properly the time zones to calculate bias
```

Comment:
did the previous method not work or why was it removed?

Reply:
The previous modification was just to allow build without error but the time zone has not compute as the IBM documentation explain. So it works on partition were time is GMT it worked but when time zone with daylight the computed time in local was buggy.

Reply:
OK, so in a nutshell it did not work previously and still does not work - right? I guess it would be helpful to update the comment so that the exact details are known. Makes it easier for others that stumble over it in the future.

Reply:
OK, I also made a mistake in my implementation. The final code is simpler. Notice : I don't how it works with hpux as the struct tz is use without to be initialized.